metadata-ingestion/src/datahub/api/entities/datajob/dataflow.py

```diff
@@ -35,11 +52,24 @@ class DataFlow:
     url: Optional[str] = None
     tags: Set[str] = field(default_factory=set)
     owners: Set[str] = field(default_factory=set)
+    platform_instance: Optional[str] = None
+    env: Optional[str] = None
```

Comment:
Previously we used `cluster` as the env value - why do we have both cluster and env now? I don't think we need this new instance variable

Reply:
Plus this is a breaking change and not deprecation. This needs to be documented in the PR doc. Plus we should not be changing the default as part of deprecation as that will cause existing deployments to have problems due to possible change of URNs.

Reply:
This isn't a breaking change yet, just a deprecation of `cluster` in favor of `env` + addition of `platform_instance`

However, we still need to make some changes:
1. the deprecation should still be noted in the PR description + our updating datahub doc
2. we should be printing a deprecation warning if folks use the old `cluster` param
3. we should throw an error if both env and cluster are provided
4. all users of this class within datahub should be updated to use `env` - I believe airflow is one, but there might be more